networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc

```diff
@@ -11,9 +11,11 @@ After you install the Kubernetes NMState Operator, you can use the Operator to o
 
 For more information about how to install the NMState Operator, see xref:../../networking/networking_operators/k8s-nmstate-about-the-k8s-nmstate-operator#k8s-nmstate-about-the-k8s-nmstate-operator[Kubernetes NMState Operator].
 
-[WARNING]
+[IMPORTANT]
 ====
-When configuring Virtual Route Forwarding (VRF) users must change their VRFs to a table ID lower than 1000 as higher than 1000 is reserved for {product-title}.
+You cannot use the `br-ex` bridge, an OVN-Kubernetes-managed Open vSwitch bridge, as the interface when configuring DNS resolvers unless you manually configured a customized `br-ex` bridge.
```

Comment:
This note is no longer specific to DNS settings. We might want to re-word it slightly to make it more generic. Something like:

You cannot provide any configuration that would modify the br-ex bridge, an OVN-Kubernetes-managed Open vSwitch bridge, unless you manually configured a customized br-ex bridge.

Reply:
Thanks. How about:

> You cannot provide any configuration that modifies the br-ex bridge, an OVN-Kubernetes-managed Open vSwitch bridge. However, you can configure a customized br-ex bridge.